android/src/androidTest/assets/cards_alphanumeric.csv

```diff
@@ -0,0 +1,8 @@
+Autographed,Condition,Brand,Year,Number,Player Name,Team,Player Position
+false,Excellent,Topps,1991,bac,Alex Fernandez,White Sox,Pitcher
+true,Mint,Topps,1974,17qefs5,Bob Stanley,Red Sox,Pitcher
+false,Very Good,Topps,1985,Ajsdh53,Vince Coleman,Cardinals,Left Field
+true,Gem Mint,TMG,1993,f5,Ken Griffey Jr.,All-Star,Center Field
+true,Excellent,Upper Deck,1993,wqeqw,Dave Hollins,Phillies,Third Base
+false,Good,Upper Deck,1990,JHG45,Tom Browning,Reds,Pitcher
+false,Near Mint,Topps,1982,GHF65jhs3,Ed Lynch,Mets,Pitcher
```

Comment:
I don't think this new file is necessary. It should be sufficient to change the number of one of the cards in `three_cards.csv`.